azure-communication-ui/azure-communication-ui/src/main/java/com/azure/android/communication/ui/redux/state/CallingState.kt

```diff
@@ -17,7 +17,10 @@ internal enum class CallingStatus {
 }
 
 internal data class CallingState(
-    val CallingStatus: CallingStatus,
+    val callingStatus: CallingStatus,
+    // due to the async nature of the CallingStatus update we need to indicate joining call
+    // until we receive CallingStatus.CONNECTING from the SDK.
+    val joinCallIsRequested: Boolean = false,
```

Comment:
Pavel, does iOS is adding same state?

Reply:
iOS has this field as a private variable on the view. On the Android we need to have access to this field from the multiple views, so it is on the Redux state

Reply:
Not sure, if we need to sync redux state Pavel. Previously, we used to follow same redux state on Android and iOS as well as feature design.

Reply:
I did sync on this topic with iOS and we agreed that it will be a difference in the implementation.